lib/ansible/config/base.yml

```diff
@@ -392,7 +392,7 @@ DEFAULT_ASK_SUDO_PASS:
   default: False
   deprecated:
     why: In favor of Ansible Become, which is a generic framework. See become_ask_pass.
-    version: "2.8"
+    version: "2.9"
```

Comment:
For modules we use `removed_in`, which is clearer. In this context `version` could be the version in which the term is deprecated, or the version in which it will be removed. The terminology pre-dates this PR, so it shouldn't block merging, but something to think about. See https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html.